packages/containerd/2001-selinux-add-DefaultLabels-helper.patch

```diff
@@ -0,0 +1,86 @@
+From 75ef4d007f7c4db0805a127d56d3347970f48a59 Mon Sep 17 00:00:00 2001
+From: Ben Cressey <bcressey@amazon.com>
+Date: Wed, 29 Apr 2020 15:10:04 +0000
+Subject: [PATCH 2001/2002] selinux: add DefaultLabels helper
```

Comment:
We discussed this being a short-lived patch because the functionality you need is present in a newer upstream version of go-selinux.  The comments below are in the context of if this patch were long-lived or intended for upstream; you may choose to address or ignore them.